src/modules/banco-proyectos/banco-proyectos.service.ts

```
@@ -0,0 +1,451 @@
+      if (metas.length === 0) {
+        return {
+          status: false,
+          message: 'No se encontraron las metas de producto especificadas',
+          error: 'Metas no encontradas',
+          data: [],
+        };
+      }
+
+      // Verificar que todas las metas tengan el mismo programa MGA
+      const codigosPrograma = metas
+        .map(meta => {
```

ESLint

[error] Replace `meta` with `(meta)`

(prettier/prettier)